Task_1/Plant.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_1
+{
+    public class Plant
+    {
+        public string Name { get; set; }
```

Comment:
Name color and height should have a limited access for setters, for example protected or private.